samples/angular/src/app/jss-graphql.service.ts

```diff
@@ -60,7 +66,7 @@ export class JssGraphQLService {
   /**
    * Executes a read query against the GraphQL endpoint
    */
-  query<T, V = R>(options: QueryOptions<V> & JssGraphQLOptions): Observable<ApolloQueryResult<T>> {
```

Comment:
`R` doesn't exists in the latest version, `EmptyObject` can be used instead of `R`